hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamer.java

```diff
@@ -1946,8 +1947,8 @@ public void testJsonKafkaDFSSource() throws Exception {
   @Test
   public void testJsonKafkaDFSSourceWithOffsets() throws Exception {
     topicName = "topic" + testNum;
-    int numRecords = 15;
-    int numPartitions = 3;
+    int numRecords = 30;
+    int numPartitions = 2;
```

Comment:
why change these values?

Reply:
this has sth to do with `org.apache.spark.streaming.kafka010.KafkaTestUtils` which only utilizes 2 partitions (probably limited by spark partitions due to 1:1 mapping). hence doing more than 2 will not give expected even records per partitions. this can be pushed as follow up to improve the test setup.

Reply:
filed the improvement ticket https://issues.apache.org/jira/browse/HUDI-6266